src/store/events.js

```diff
@@ -1,45 +1,47 @@
-import eventService from '../services/eventService'
-import Vue from 'vue'
+import eventService from "../services/eventService";
+import Vue from "vue";
 
 const events = {
-    namespaced: true,
-    state: {
-        general: [],
-        local: {}
+  namespaced: true,
+  state: {
+    general: [],
+    local: {}
+  },
+  mutations: {
+    setGeneralEvents(state, events) {
+      state.general = [...state.general, ...events];
     },
-    mutations: {
-        setGeneralEvents(state, events) {
-            state.general = [...state.general, ...events]
-        },
-        setLocalEvent(state, event) {
-            state.local = event
-        }
+    setLocalEvent(state, event) {
+      state.local = event;
+    }
+  },
+  actions: {
+    // "QUERY" - makes some form of request to get data from the DB
+    queryAllEvents({ commit }) {
+      return eventService
+        .getAllEventData()
+        .then(data => {
+          commit("setGeneralEvents", data.data);
+        })
+        .catch(err => {
+          Vue.toasted.global.errorMessage({
+            message: err
+          });
+        });
     },
-    actions: {
-        // "QUERY" - makes some form of request to get data from the DB
-        queryAllEvents({ commit }) {
-            return eventService.getAllEventData()
-            .then((data) => {
-                commit('setGeneralEvents', data.data)
-            })
-            .catch((err) => {
-                Vue.toasted.global.errorMessage({
-                    message: err
-                })
-            })
-        },
-        queryLocalEvent({ commit }, id) {
-            return eventService.getEventData({ id })
-            .then((data) => {
-                commit('setLocalEvent', data.data)
-            })
-            .catch((err) => {
-                Vue.toasted.global.errorMessage({
-                    message: err
-                })
-            })
-        }
+    queryLocalEvent({ commit }, id) {
+      return eventService
+        .getEventData({ id })
+        .then(data => {
+          commit("setLocalEvent", data.data);
```

Comment:
```suggestion
        .then(response => {
          commit("setLocalEvent", response.data);
```

Reply:
Ok this isn't a necessary change mate

Reply:
It's a mistake on my end - if you could change it in this PR it would be muuuch appreciated ❤️
Naming inconsistencies bother me

Reply:
alriiiight 👍